opencensus/proto/export/export.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "io.opencensus.export";
+option java_outer_classname = "ExportProto";
+
+option go_package = "github.com/census-instrumentation/opencensus-proto/gen-go/exportproto";
+
+package export;
+
+import "trace/trace.proto";
+import "stats/metrics/metrics.proto";
+
+service Export {
+  rpc Export (ExportRequest) returns (ExportResponse) {}
+}
+
+message ExportRequest {
+  repeated opencensus.proto.trace.Span spans = 1;
+
+  repeated opencensus.proto.stats.metrics.Metric metrics = 2;
```

Comment:
Why not have separate RPCs or even separate services for spans and metrics? This would allow us more visibility into the two quite distinct operations.

Reply:
I haven't separated them for the connivence of the exporter implementations. I expect the exporter to batch some metrics and spans and upload once a while. If we want to have more visibility into metrics vs spans calls, we should split.

Reply:
I don't really see how this is more convenient for exporter implementations than having separate RPCs. Maybe I am not imagining the same implementation but today the exporters are pretty different in how they batch (stats vs. trace). And in any case I would say that the observability advantages of having separate RPCs or even separate services are more important.